apps/nextra/components/landing/components/Accordion.tsx

```diff
@@ -0,0 +1,68 @@
+import * as AccordionPrimitive from "@radix-ui/react-accordion";
+import { Button } from "./Button";
+import { IconArrowTopRight } from "./Icons";
+import { Link } from "nextra-theme-docs";
+import { cn } from "utils/cn";
+
+export interface AccordionItem {
+  id: string;
+  label: string;
+  description: string;
+  href: string;
+}
+
+export interface AccordionProps {
+  value: string;
+  onChange: (value: string) => void;
+  items: AccordionItem[];
+  HeadingLevel: "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
```

Comment:
I'm kind of confused how this is used? Maybe you can educate me

Reply:
Each section of the `Accordion` has a heading label. However, since the component is reusable and can be placed anywhere in the document, the appropriate heading level may vary. For example, if we had an `h1` that said "Move Examples" and then had this accordion right below it, we might want to set the heading level for each accordion section to `h2`. In our current case, there's an `h2` at the top of this Move section, so we tell the `Accordion` component to render `h3`s for its headings. Hopefully that clarifies things, but lmk if not.